lib/db/db_sqlite.rb

```diff
@@ -0,0 +1,85 @@
+require 'db/db'
+require 'sqlite3'
+
+class DBSQLite < DB
+
+  def initialize(sqlite, table, columns)
```

Comment:
Does that mean UI requirements force the design of the database? Or the database structure force the design of the UI?
Either case is not ideal. The UI should be the UI, and the database and it's structure is something completely different.